client/src/components/atoms/StepNumber.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FunctionComponent } from "react";
+
+interface StepNumberProps {
+  stepNumber: number;
+  isSelectedStep: boolean;
+}
+
+const StepNumber: FunctionComponent<StepNumberProps> = (
+  props: StepNumberProps
+) => {
+  return (
+    <div
+      className={
+        "step-number " + (props.isSelectedStep ? "selected-step" : "")
+      }
+    >
+      <h3>
+        {props.stepNumber.toLocaleString("en-US", {
```

Comment:
Just wondering why toLocaleString is necessary

Reply:
Using `toLocaleString` to format  the number `props.stepNumber` into a number with 2 digits